apps/condo/domains/common/schema/fields.js

```diff
@@ -131,6 +132,34 @@ const MONEY_AMOUNT_FIELD = {
     },
 }
 
+const POSITIVE_MONEY_AMOUNT_FIELD = {
+    ...MONEY_AMOUNT_FIELD,
+    hooks: {
+        validateInput: ({ resolvedData, addFieldValidationError, fieldPath, listKey }) => {
```

Comment:
Probably it's better to create a custom field type. You can accidentally overwrite the validated input logic

Reply:
Created signedDecimal field for this purpose